Cron staging runtime integration failing

<!-- ⚠️ If you do not respect this template, your issue will be closed -->
<!-- ⚠️ Make sure to browse the opened and closed issues -->

### Information

- **Qiskit IBMQ Provider version**:
- **Python version**:
- **Operating system**:

### What is the current behavior?
[Cron staging runtime integration](https://github.com/Qiskit-Partners/qiskit-ibm/runs/3502696588?check_suite_focus=true) tests have been running for more than 6 hours and failing.


### Steps to reproduce the problem



### What is the expected behavior?



### Suggested solutions

New issue https://github.com/Qiskit-Partners/qiskit-ibm/runs/3659295749?check_suite_focus=true
```
python -m unittest -v test/ibm/runtime/test_runtime_integration.py
ibm_provider._discover_remote_backends:WARNING:2021-09-21 04:30:51,912: Remote backend "***" for provider <IBMProvider(hub='rte-test', group='lp-all', project='renierm')> could not be instantiated due to an invalid config: Traceback (most recent call last):
  File "/Users/runner/work/qiskit-ibm/qiskit-ibm/qiskit_ibm/ibm_provider.py", line 472, in _discover_remote_backends
    config = PulseBackendConfiguration.from_dict(raw_config)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/qiskit/providers/models/backendconfiguration.py", line 692, in from_dict
    input_uchannels = in_data.pop("u_channel_lo")
KeyError: 'u_channel_lo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/runner/work/qiskit-ibm/qiskit-ibm/qiskit_ibm/ibm_provider.py", line 474, in _discover_remote_backends
    config = QasmBackendConfiguration.from_dict(raw_config)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/qiskit/providers/models/backendconfiguration.py", line 402, in from_dict
    return cls(**in_data)
TypeError: __init__() missing 2 required positional arguments: 'local' and 'conditional'

ERROR

======================================================================
ERROR: setUpClass (test.ibm.runtime.test_runtime_integration.TestRuntimeIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/qiskit-ibm/qiskit-ibm/test/decorators.py", line 70, in _wrapper
    return func(obj, *args, **kwargs)
  File "/Users/runner/work/qiskit-ibm/qiskit-ibm/test/decorators.py", line 227, in _wrapper
    _backend = _get_backend(qe_token=kwargs.pop('qe_token'),
  File "/Users/runner/work/qiskit-ibm/qiskit-ibm/test/decorators.py", line 256, in _get_backend
Error:     raise Exception('Unable to find a suitable backend.')
Exception: Unable to find a suitable backend.

----------------------------------------------------------------------
Ran 0 tests in 16.541s

FAILED (errors=1)
make: *** [runtime_integration] Error 1
Error: Process completed with exit code 2.
```

--- a/.github/workflows/cron-prod.yml
+++ b/.github/workflows/cron-prod.yml
@@ -1,0 +1,28 @@
+name: Cron-prod
+on:
+  schedule:
+    - cron: '0 4 * * *'
+jobs:
+  runtime-integration:
+    name: runtime-integration
+    runs-on: macOS-latest
+    env:
+      QISKIT_IBM_API_TOKEN: ${{ secrets.QISKIT_IBM_API_TOKEN }}
+      QISKIT_IBM_API_URL: ${{ secrets.QISKIT_IBM_API_URL }}
+      QISKIT_IBM_RUNTIME_DEVICE: ${{ secrets.QISKIT_IBM_RUNTIME_DEVICE }}
+      LOG_LEVEL: DEBUG
+      STREAM_LOG: True
+      QISKIT_IN_PARALLEL: True
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install Deps
+        run: |
+          python -m pip install --upgrade pip
+          pip install -c constraints.txt -e .
+          pip install -U -c constraints.txt -r requirements-dev.txt
+      - name: Run Tests
+        run: make runtime_integration

--- a/test/ibm/runtime/test_runtime_integration.py
+++ b/test/ibm/runtime/test_runtime_integration.py
@@ -38,10 +38,6 @@ from ...proxy_server import MockProxyServer, use_proxies
 from .utils import SerializableClass, SerializableClassDecoder, get_complex_types
 
 
-@unittest.skipIf(
-    not os.environ.get('QISKIT_IBM_USE_STAGING_CREDENTIALS', ''),
-    "Only runs on staging"
-)
 class TestRuntimeIntegration(IBMTestCase):
     """Integration tests for runtime modules."""
 
@@ -152,14 +148,11 @@ def main(backend, user_messenger, **kwargs):
     def test_upload_program(self):
         """Test uploading a program."""
         max_execution_time = 3000
-        is_public = True
-        program_id = self._upload_program(max_execution_time=max_execution_time,
-                                          is_public=is_public)
+        program_id = self._upload_program(max_execution_time=max_execution_time)
         self.assertTrue(program_id)
         program = self.provider.runtime.program(program_id)
         self.assertTrue(program)
         self.assertEqual(max_execution_time, program.max_execution_time)
-        self.assertEqual(program.is_public, is_public)
 
     def test_upload_program_file(self):
         """Test uploading a program using a file."""
@@ -173,6 +166,26 @@ def main(backend, user_messenger, **kwargs):
         program = self.provider.runtime.program(program_id)
         self.assertTrue(program)
 
+    @unittest.skipIf(
+        not os.environ.get('QISKIT_IBM_USE_STAGING_CREDENTIALS', ''),
+        "Only runs on staging"
+    )
+    def test_upload_public_program(self):
+        """Test uploading a public program."""
+        max_execution_time = 3000
+        is_public = True
+        program_id = self._upload_program(max_execution_time=max_execution_time,
+                                          is_public=is_public)
+        self.assertTrue(program_id)
+        program = self.provider.runtime.program(program_id)
+        self.assertTrue(program)
+        self.assertEqual(max_execution_time, program.max_execution_time)
+        self.assertEqual(program.is_public, is_public)
+
+    @unittest.skipIf(
+        not os.environ.get('QISKIT_IBM_USE_STAGING_CREDENTIALS', ''),
+        "Only runs on staging"
+    )
     def test_set_visibility(self):
         """Test setting the visibility of a program."""
         program_id = self._upload_program()

